bazelrio/dependencies/googletest/1_9_0_5_437e100_1/deps.bzl

```diff
@@ -0,0 +1,33 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
I would rather have the example pull it in raw and build from source instead of as a precompiled library

1. Its a small library and shows how you would pull in third party source without having to make a copy locally, or precompile it somewhere else, 
2. You get full control over gtest. You can use their `gtest_main` lib rather than having to make your own `main.cc` like allwpilib is doing
3. You have full control over the gtest version, rather than being locked into one of the ones allwpilib has precompiled